Yoast/wpseo-woocommerce
# Replace the HelpScout beacon

## Summary

This PR can be summarized in the following changelog entry:

* Replaces the HelpScout beacon on the WooCommerce SEO admin page.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to the WooCommerce admin page and see that HelpScout is working (and shows articles that are specifically relevant to WooCommerce).

Fixes #438
